devnet/README.md

```diff
@@ -23,6 +23,40 @@ docker-compose down
 
 > **_NOTE:_**  The devnet is not stateful, so subsequent runs will start from the genesis block.
 
+## Running Cross Client Devnet
```

Comment:
do these instructions make sense to put here or should they go somewhere in docs/ ? since this devnet is for gossamer-only

Reply:
The cross client devnet will live here as well.  Ideally we'd move this whole thing out of the gossamer repo at some point, so I don't mind it living here for now.